portal/eproms_substudy_tailored_content/src/style/fonts/fontawesome-webfont.svg

```diff
@@ -0,0 +1,2671 @@
+<?xml version="1.0" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd" >
+<svg>
+<metadata>
+Created by FontForge 20120731 at Mon Oct 24 17:37:40 2016
+ By ,,,
+Copyright Dave Gandy 2016. All rights reserved.
+</metadata>
+<defs>
+<font id="FontAwesome" horiz-adv-x="1536" >
```

Comment:
Is there a version of this hosted on a CDN we should use- either directly link to, or download as part of the front-end build process?

Reply:
thank you, fixed